lib/std/zig/system/linux.zig

```diff
@@ -374,6 +374,13 @@ fn CpuinfoParser(comptime impl: anytype) type {
     };
 }
 
+inline fn getCpuFeature(comptime feat_reg: []const u8) u64 {
+    return asm volatile ("mrs %[x], " ++ feat_reg
+                         : [ret] "=r" (-> u64),
+                         : [x] "r" (feat_reg),
```

Comment:
I think that should be what you want:

```suggestion
    return asm volatile ("mrs %[ret], " ++ feat_reg
                         : [ret] "=r" (-> u64),
```

Allocates a register for the result of `mrs`, uses that for the result expression value.